packages/autoconf/2.71/0001-Update-autoconf-and-add-new-version.patch

```diff
@@ -0,0 +1,35 @@
+From a09e46a5413066896e7a18c0032b5e2da35df1f9 Mon Sep 17 00:00:00 2001
```

Comment:
This file seems to have been added in error

Reply:
Fixed both.